lstchain/reco/r0_to_dl1.py

```diff
@@ -221,10 +219,31 @@ def r0_to_dl1(
     config = replace_config(standard_config, custom_config)
 
     custom_calibration = config["custom_calibration"]
-    gain_selector = load_gain_selector_from_config(config)
+
+    source_config = {
+        "LSTEventSource": {
+            "allowed_tels": [1],
+            "EventTimeCalculator": {
+                "ucts_t0_dragon": ucts_t0_dragon,
+                "dragon_counter0": dragon_counter0,
+                "ucts_t0_tib": ucts_t0_tib,
+                "tib_counter0": tib_counter0
+            },
+            "PointingSource":{
+                "drive_report_path": pointing_file_path
+            },
+            "LSTR0Corrections":{
+                "calibrate_ff_ped": False,
```

Comment:
This should be calibrate_ff_and_ped   - but I tried it and it is not recognized anyway, with ctapipe_io_lst 0.6.1